hollow/src/main/java/com/netflix/hollow/api/consumer/metrics/AbstractRefreshMetricsListener.java

```diff
@@ -0,0 +1,178 @@
+package com.netflix.hollow.api.consumer.metrics;
+
+import com.netflix.hollow.api.consumer.HollowConsumer;
+import com.netflix.hollow.api.custom.HollowAPI;
+import com.netflix.hollow.core.read.engine.HollowReadStateEngine;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Objects;
+
+public class AbstractRefreshMetricsListener extends HollowConsumer.AbstractRefreshListener  {
```

Comment:
I have some thoughts on "Refresh" as the verb/noun we use and why (for better or worse) I started to introduce different terminology in the Cinder Dashboard.

I landed on "consumer state update" which I'm not in love with, but is consistent with some other terminology in Hollow ("states"). The problem I have with "refresh" is that it implies an already loaded state and so it can be confusing that the initial load of data is also a refresh.

That said, refresh terminology is already baked into our class names. So the decision should be if we're committed to eventually renaming things like `RefreshListener`; otherwise introducing a *consumer state update* as a synonym for *refresh* just makes things worse.

Reply:
I think that might have something to do with the fact that I didn't come across ConsumerStateUpdate until you mentioned it in one of the PR comments :P Your comment makes perfect sense to me, and I'll look into this further, but worried about the blast radius of renaming refresh to state everywhere.

Reply:
Decided to keep the current terminology for now to limit the scope of this change. I like the idea of calling it state update, but renaming refresh to state update has a wider blast radius.  I think it would be better suited for a different PR.